package.json

```diff
@@ -26,6 +26,7 @@
     "co-mocha": "^1.2.0",
     "commander": "^2.9.0",
     "coveralls": "^2.11.16",
+    "@sugarcrm/eslint-config-sugarcrm": "git+https://github.com/sugarcrm/javascript",
```

Comment:
if we can publish the package today, would be great to just add this one from npm directly